build.sbt

```diff
@@ -833,6 +833,7 @@ lazy val docs = project
   .dependsOn(scalameta.jvm)
   .settings(
     sharedSettings,
+    crossScalaVersions := List(LatestScala213),
```

Comment:
is this for completeness? or without it, no version of docs will be built?

Reply:
By default it has three Scala versions if I recall correctly, and it would fail to resolve mdoc for Scala 2.11 or something like that, so I narrowed it to one.